src/views/ModelPlan/ReadOnly/GeneralCharacteristics/index.tsx

```diff
@@ -172,111 +213,151 @@ const ReadOnlyGeneralCharacteristics = ({
+                copy={translateBooleanOrNull(collectPlanBids)}
+                notes={collectPlanBidsNote}
+              />
+            )}
 
-            <ReadOnlySection
-              heading={t('manageEnrollment')}
-              copy={translateBooleanOrNull(managePartCDEnrollment)}
-              notes={managePartCDEnrollmentNote}
-            />
+            {checkGroupMap(
+              isViewingFilteredView,
+              filteredQuestions,
+              'manageEnrollment',
+              <ReadOnlySection
+                heading={t('manageEnrollment')}
+                copy={translateBooleanOrNull(managePartCDEnrollment)}
+                notes={managePartCDEnrollmentNote}
+              />
+            )}
 
-            <ReadOnlySection
-              heading={t('updatedContract')}
-              copy={translateBooleanOrNull(planContractUpdated)}
-              notes={planContractUpdatedNote}
-            />
+            {checkGroupMap(
+              isViewingFilteredView,
+              filteredQuestions,
+              'updatedContract',
+              <ReadOnlySection
+                heading={t('updatedContract')}
+                copy={translateBooleanOrNull(planContractUpdated)}
+                notes={planContractUpdatedNote}
+              />
+            )}
           </>
         )}
       </div>
 
-      <div className="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light">
-        <div className="desktop:display-flex flex-justify">
-          <div className="desktop:width-card-lg">
-            <ReadOnlySection
-              heading={t('careCoordination')}
-              copy={translateBooleanOrNull(careCoordinationInvolved)}
-            />
-          </div>
-          {careCoordinationInvolved && (
-            <div className="desktop:width-card-lg">
-              <ReadOnlySection
-                heading={h('howSo')}
-                copy={careCoordinationInvolvedDescription}
-              />
-            </div>
-          )}
-        </div>
+      <div
+        className={`${
+          isViewingFilteredView
+            ? ''
+            : 'margin-bottom-4 border-bottom-1px border-base-light padding-bottom-2'
+        }`}
+      >
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'careCoordination',
+          <SideBySideReadOnlySection
+            firstSection={{
+              heading: t('careCoordination'),
+              copy: translateBooleanOrNull(careCoordinationInvolved)
+            }}
+            secondSection={
+              careCoordinationInvolved === true && {
+                heading: h('howSo'),
+                copy: careCoordinationInvolvedDescription
+              }
+            }
+          />
+        )}
         {careCoordinationInvolvedNote && (
           <ReadOnlySection
             heading={t('basics:notes')}
             copy={careCoordinationInvolvedNote}
           />
         )}
```

Comment:
Another instance where this component will always render in every view if a note exists